lib/pdc/resource/associations/has_many.rb

```diff
@@ -0,0 +1,23 @@
+require 'pdc/resource/associations/association'
+
+module PDC::Resource
+  module Associations
+    class HasMany < Association
+      def initialize(*args)
+        super
+        @options.reverse_merge!(uri: "#{parent_path}/:#{foreign_key}/#{@name}/(:#{primary_key})")
```

Comment:
Seems even we don't set an uri when init the association, this default uri can't work.
If so I prefer to remove this line

Reply:
I think if u don't set the uri, this default one will be used. Maybe in the future we will consider to change the API in this way :)

Reply:
OK, got it.
I think we need a comment here to let people know they should add "uri" when they init the association because this default setting can't work currently.